app/Jobs/PostMark/ProcessPostmarkWebhook.php

```diff
@@ -374,29 +380,31 @@ public function getBounceId(string $message_id): ?int
 
     private function fetchMessage(): array
     {
-        if(strlen($this->request['MessageID']) < 1) {
+        if (strlen($this->request['MessageID']) < 1) {
             return $this->default_response;
         }
 
         try {
 
-            $postmark = new PostmarkClient(config('services.postmark.token'));
+            $postmark_secret = $this->company->settings->postmark_secret ?? config('services.postmark.token');
```

Comment:
This will always resolve settings->postmark_secret as it is an empty string by default

Reply:
fixed